components/button/button.css

```diff
@@ -0,0 +1,91 @@
+.btn {
+  box-sizing: border-box;
+  border: none;
+  padding: 0.6rem 1rem;
```

Comment:
for size less than 1rem you can use px, rest all is fine 👍

Reply:
Sure